packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -20,6 +20,7 @@ spec:
           image: "{{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}:{{ .Values.images.fluentbit.tag }}"
           {{- if .Values.global.seLinux.enabled }}
           securityContext:
+            privileged: true
```

Comment:
As I understand it, we don't want to run any privileged containers. It is incompatible with SELinux controls.

Reply:
Agreed with Caleb, what's the motivation behind this change?